_tests/integration/mac_test.go

```diff
@@ -113,9 +109,6 @@ configs:
               run_if: '{{getenv "SSH_RSA_PRIVATE_KEY" | ne ""}}'
           - git-clone@%s: {}
           - cache-pull@%s: {}
-          - script@%s:
```

Comment:
Should not we add description to macOS wfs too? It is almost the same as the ios description.

Reply:
We do not have a macOS section in our getting started guides so even if the workflows are similar we could not link to anything useful. This is why I decided to not to change the macOS default workflow description.